src/EventGridEmulator/Properties/launchSettings.json

```diff
@@ -1,13 +1,21 @@
-﻿{
+{
   "profiles": {
     "https": {
       "commandName": "Project",
-      "dotnetRunMessages": false,
-      "launchBrowser": false,
-      "applicationUrl": "https://localhost:5000",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
-      }
+      },
+      "dotnetRunMessages": false,
+      "applicationUrl": "https://localhost:5000"
+    },
+    "Docker": {
+      "commandName": "Docker",
+      "launchUrl": "{Scheme}://{ServiceHost}:{ServicePort}",
+      "environmentVariables": {
+        "ASPNETCORE_URLS": "https://+:443;http://+:80"
```

Comment:
Does that work with http? I feel like it will override 8080 in the docker file and 80 is a privileged port, so maybe sticking to 8080 is better